.github/workflows/kindIntegTest.yml

```diff
@@ -47,62 +47,177 @@ jobs:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/checkout@v2
+        if: github.event_name != 'pull_request'
+      - uses: ./.github/actions/run-integ-test
+        with:
+          integration_test: ${{ matrix.integration_test }}
+      - name: Archive k8s logs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: k8s-logs-${{ matrix.integration_test }}-${{ matrix.version }}
+          path: ./build/kubectl_dump
+
+  kind_dse_tests:
+    needs: build_docker_images
+    strategy:
+      matrix:
+        version:
+        - "6.8.29"
+        integration_test:
+        #- delete_node_lost_readiness #TODO: Fix in MC-356
+        - additional_seeds
+        include:
+          - version: "6.8.29"
+            serverImage: datastax/dse-mgmtapi-6_8:6.8.29-jdk8
+            serverType: dse
+    runs-on: ubuntu-latest
+    env:
+      CGO_ENABLED: 0
+      M_INTEG_DIR: ${{ matrix.integration_test }}
+      M_SERVER_VERSION: ${{ matrix.version }}
+      M_SERVER_TYPE: dse
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'pull_request'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: actions/checkout@v2
+        if: github.event_name != 'pull_request'
+      - uses: ./.github/actions/run-integ-test
+        with:
+          integration_test: ${{ matrix.integration_test }}
+      - name: Archive k8s logs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v2
+        with:
+          name: k8s-logs-${{ matrix.integration_test }}-dse-${{ matrix.version }}
+          path: ./build/kubectl_dump
+
+  kind_int_tests:
+    needs: build_docker_images
+    strategy:
+      matrix:
+        version:
+        - "3.11.14"
+        - "4.0.7"
+        - "6.8.29"
         integration_test:
         # Single worker tests:
         - additional_serviceoptions
         - additional_volumes
         # - delete_node_terminated_container # This does not test any operator behavior
         - podspec_simple
         # - smoke_test_oss # See next job
-        - smoke_test_dse
+        # - smoke_test_dse # See next job
         - terminate
         - timeout_prestop_termination
-        - upgrade_operator
+        # - upgrade_operator # See above job
         - webhook_validation
         # Three worker tests:
-        - canary_upgrade
+        # - canary_upgrade # See above job
         - config_change_condition
-        - cdc_successful
-        # - delete_node_lost_readiness # DSE specific behavior
+        # - cdc_successful # CDC is OSS only , see above jobs
+        # - delete_node_lost_readiness # DSE specific behavior see above job
         - host_network
         - internode-encryption-generated
         #- no_infinite_reconcile # smoke_test_* should take care of this
         - node_replace
         - nodeport_service
-        - rolling_restart
+        #- rolling_restart #TODO: Fix in MC-356
```

Comment:
No MC ticket references, ticket to fix missing issues should be in the Github repo.